### ISSUE TITLE
gpg - jobs landing pages

### ISSUE THREAD

**Issue by octavioamu** [gitcoinco/web#8658]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a User I want to have earn, learn, connect, fund landing pages so I can understand gitcoin potential.
JTBD: jobs to be done

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: feature

### Current Behavior
[comment]: # (Describe what actually happened.)
no jtbd pages 
### Expected Behavior
[comment]: # (Describe what you expected to happen.)
jbbd pages exist and can be access by logged/not logged users from the menu 
### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

**Figma** file https://www.figma.com/file/pOEKRzmB6pcbQzBp04S0l4/Gitcoin.co?node-id=292%3A6285
**Copy** 
https://docs.google.com/document/d/1yOc32tRNDKCAynNcixhIKgtv9pv6LcrmlR4ueC_x5Wk/edit (each page copy link inside)

**gpg/jtbd-landings** will be the integration branch to/from **master** the progressive prs for each work will be from/to this PR

- [gpg/jtbd-landings](https://github.com/gitcoinco/web/pull/8706)
    - [gpg/jtbd-backend](https://github.com/gitcoinco/web/pull/8693)  _2 days work_  	
    - [gpg/jtbd-earn](https://github.com/gitcoinco/web/pull/8703)  _3 days_
    - [gpg/jtbd-learn](https://github.com/gitcoinco/web/pull/8711)  _2 days_
    - [gpg/jtbd-connect](https://github.com/gitcoinco/web/pull/8704) _2 days_  	
    - [gpg/jtbd-fund](https://github.com/gitcoinco/web/pull/8709) _2 days_
    - gpg/jtbd-mautic _1 days_

---------------

**gpg/jtbd-backend** _2 days work_
- Create the queries for each data query on the designed pages in jsonStores, queries need to be limited on used values and amounts of each based on designs (ex just query 3 grants and fields title, description, logo, etc)
- Add the queries to create_page_cache and add it to the command so will be ran on the crontab together with others queries and generate a jsonstore entry for each.
- Create the views def for each of the pages querieing the jsonStores by view/key and use cache for those (see `context['last_posts'] = cache.get_or_set('last_posts', fetchPost, 5000)` for reference)

**gpg/jtbd-earn** _3 days_
- Create collapsible using a custom class to overwrite the styles
- Extend bootstrap collapsible to change the image outside of the collapsible
- Html changes using bootstrap as much of possible to look as the figma, avoid recreating class and styles if not needed. 

**gpg/jtbd-learn** _2 days_
**gpg/jtbd-connect** _2 days_
**gpg/jtbd-fund** _2 days_
- reuse collapsible 
- Html changes using bootstrap as much of possible to look as the figma, avoid recreating class and styles if not needed. 

**gpg/jtbd-mautic** _1 days_
- We plan to use sdk events to capture anonymous users until they get converted to logged users.
- Sync with Octavio to learn about mautic.
- Sync with mkt to align what they want to collect.
- Marketing to tell us what they will do on the mautic side. 

---------------
### Total 12 days of work
**Helps that can be done in parallel are:** 
collapsible front end
pages
mautic depending on mkt requests can be done in parallel or need to wait until everything is done but mostly like to campaings be build based on hit pages logic on mautic side and not code. 

**Menu changes are not part of this task**

### Data Requirements
[comment]: # (How will we measure the success of this feature? What kind of tracking is needed for this feature (clicks, impressions, flag)?)
see **gpg/jtbd-mautic**

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)

**Comment by chibie** [comment]
@PixelantDesign @frankchen07 here are some questions,

* Which bounty data should show in jtbd/earn "Earn through hackathons and bounties"? those related to hackathons or not?
* Which grant should show in jtbd/earn “Earn through grants”?
* Which projects should show in jtbd/connect "Successful projects". random, latest, or most funded?
* Meet passionate builders in jtbd/fund… funders, hackathon participants, or leaderboard?

**Comment by frankchen07** [comment]
@chibie 

- might've missed out on the final decision for this one, but for top earners, maybe we do it monthly, Kyle made the point that perhaps we want to do it monthly so that there's a rotation of characters, not just the same 2-3 people. If Graham had a point against this one that I missed though, all good, we can keep the cadence to all.

- bounties not related to hackathons (highest values) - let's confirm with @PixelantDesign to get her final opinion.

- undiscovered gems for grants 👌

- projects would be the more successful projects, but this might be difficult to determine simply using db data. Probably a winning submission with at least 1 team member, if we go that route.

- passionate builders, this could be a good place to feature past Grant successes, something like grants that got off the ground from grant -> legit project / company (if we pull that data more manually), or if something more automatic, then top leaderboard for grant amount contributed in the last round could work too.

- will follow up with Matt on the earn testimonial quote.

**Comment by frankchen07** [comment]
also 👏  for octavio & the engs for breaking up the epic!

**Comment by PixelantDesign** [comment]
- Top Earners question
Agree with @frankchen07 on Top Earners by month (year is too long not enough change, week is too short).

- Bounty data on earn page
Hackathons section - show hackathon related
Bounties section - show most recent highest value bounty _not in a hackathon_ so that there is variety between the two sections.

- Which grant should show in jtbd/earn “Earn through grants”
This section won't be able to be dynamic as @frankchen07 mentioned. I used Wallet Connect since Scott considers them a top project. Choosing a project that went fr hacks>grants would make sense as well.

- Which projects should show in jtbd/connect "Successful projects". random or latest hackathons?~
Please show SWIVEL and EPNS as shown in mockups and selected by Angela and Sachin.

- Meet passionate builders in jtbd/fund… top grantees by leaderboard or top earners, but let me confirm that.

**Comment by PixelantDesign** [comment]
@chibie lets use these for JTBD Grants Meet passionate builders:

https://gitcoin.co/austintgriffith/grants
https://gitcoin.co/koeppelmann/grants
https://gitcoin.co/poapxyz/grants
https://gitcoin.co/syroamst/grants

**Comment by PixelantDesign** [comment]
@chibie 

Chibie for the Fund Page Scott is working on a good testimonial, please put in a placeholder for now. Thanks!

cc @ceresstation

**Comment by PixelantDesign** [comment]
Here is where I'm capturing testimonials https://docs.google.com/document/d/1DejQPJ2eyxlB96bdaaGNKB4mm97TAQymM-oLiO3T9OE/edit?usp=sharing

**Comment by frankchen07** [comment]
some production feedback for the **learn** page: let's add the quests CTA back as per data evidence, and closing the loop on owocki's request

![Screen Shot 2021-04-08 at 14 41 21](https://user-images.githubusercontent.com/7516920/114101791-170c1800-987b-11eb-9b5a-c166ae5856d8.png)
![Screen Shot 2021-04-08 at 14 59 06](https://user-images.githubusercontent.com/7516920/114101794-18d5db80-987b-11eb-8da4-cebd3ee0c29f.png)

**Comment by frankchen07** [comment]
for green nft hackathon on the EARN page:

there's some spacing inconsistencies

![Screen Shot 2021-04-08 at 15 05 53](https://user-images.githubusercontent.com/7516920/114102341-fd1f0500-987b-11eb-8334-a4cd27e47c94.png)

**Comment by frankchen07** [comment]
- [ ] on the **EARN** page, the tags make it look like this section is broken:

![Screen Shot 2021-04-08 at 15 07 07](https://user-images.githubusercontent.com/7516920/114102424-2344a500-987c-11eb-9c4f-97b874bc9cec.png)

- [ ] on the **EARN** page, for the grants section, how can we make this look "more like a grant and less like an advertisement?" I got the feeling I was almost looking at a walletconnect ad.

![Screen Shot 2021-04-08 at 15 08 11](https://user-images.githubusercontent.com/7516920/114102535-525b1680-987c-11eb-94a7-8890fb7a1b33.png)

**Comment by frankchen07** [comment]
on the **CONNECT** page, we have an "apply to kernel" CTA in the middle of the page, which is inconsistent with the other JTBD landing pages. Is this intended and what we want? I would argue that for our connect JTBD, we do want to funnel them towards a "connect" action (via Kernel CTA at the bottom), but they can get to the apply form from the bottom CTA, so we don't have multiple pathways to Kernel.

![Screen Shot 2021-04-08 at 15 14 52](https://user-images.githubusercontent.com/7516920/114103050-491e7980-987d-11eb-97b4-dc365fdc8277.png)

We can also see that the Kernel CTA at the bottom of the page essentially leads them to the same place to apply.

![Screen Shot 2021-04-08 at 15 17 03](https://user-images.githubusercontent.com/7516920/114103200-8a168e00-987d-11eb-86c7-66fa1ed02814.png)

**Comment by frankchen07** [comment]
- [ ] on the **FUND** page, is there a better way to format the text so it looks less awkward, or is there a visual that shows it better?

![Screen Shot 2021-04-08 at 15 18 36](https://user-images.githubusercontent.com/7516920/114103313-c944df00-987d-11eb-9a3e-fbdaef3e0e97.png)

- [ ] on the **FUND** page, different box size

![Screen Shot 2021-04-08 at 15 19 27](https://user-images.githubusercontent.com/7516920/114103355-da8deb80-987d-11eb-9894-c6bf73b74f1c.png)

**Comment by owocki** [comment]
feedback
https://gitcoin.co/earn
- why is there only one grant shown on this page?
https://gitcoin.co/learn
- quests should be linked as a CTA

**Comment by matthewcarano** [comment]
feedback
https://gitcoin.co/earn

<img width="789" alt="Screen Shot 2021-04-09 at 5 38 47 PM" src="https://user-images.githubusercontent.com/30149603/114243412-7f7df680-995a-11eb-8137-b7a8e9ad3903.png">

- Consider changing the header padding to enable:

Break free from your 9–5
to work on open source
projects that matter.

____________________________________

<img width="1122" alt="Screen Shot 2021-04-09 at 5 30 35 PM" src="https://user-images.githubusercontent.com/30149603/114242858-93752880-9959-11eb-9a63-1c6bf8187671.png">

- In the footer, the CTAs capitalization should be consistent with the CTA in the hero. I recommend we capitalize the first word, lower case the rest for all CTAs. But ultimately this is a design choice.

- There is double copy in the "Earn in hackathons" box. The correct copy is: "Earn while working on top open source projects, with great leaders in crypto."

**Comment by matthewcarano** [comment]
feedback
https://gitcoin.co/learn

<img width="1131" alt="Screen Shot 2021-04-09 at 5 47 55 PM" src="https://user-images.githubusercontent.com/30149603/114244071-ca4c3e00-995b-11eb-8a7e-272f54643898.png">

- Consider giving the text in this quote more horizontal space, and breaking it into 2 paragraphs. Also, please add a "." to the end of the quote.

___________________________

<img width="1130" alt="Screen Shot 2021-04-09 at 5 47 09 PM" src="https://user-images.githubusercontent.com/30149603/114244254-2020e600-995c-11eb-8020-ecf8025601bc.png">

- Please match CTA sentence case. (Explore Kernel will remain capitalized).

**Comment by matthewcarano** [comment]
feedback
https://gitcoin.co/connect

<img width="1064" alt="Screen Shot 2021-04-09 at 5 56 27 PM" src="https://user-images.githubusercontent.com/30149603/114244700-f87e4d80-995c-11eb-9a89-b670f65ae80a.png">

- This is the Hackathons section. Is it possible to vertically align the text areas (subhead, date, sponsored by)

_______________________________

<img width="1160" alt="Screen Shot 2021-04-09 at 5 59 47 PM" src="https://user-images.githubusercontent.com/30149603/114244941-72163b80-995d-11eb-8a50-59cd1122642c.png">

- Please change copy to "Meet the Gitcoin community in Discord."

______________________________

- Please use sentence case in the bottom CTA "Explore hackathons"

**Comment by matthewcarano** [comment]
feedback
https://gitcoin.co/fund

<img width="1119" alt="Screen Shot 2021-04-09 at 6 04 45 PM" src="https://user-images.githubusercontent.com/30149603/114245281-2adc7a80-995e-11eb-9013-cab4339dc9c2.png">

- Please add "." at the end of all sentences.

__________________________

- Please use sentence case in the footer CTAs.

**Comment by PixelantDesign** [comment]
**Earn Page**

@chibie we are missing the dynamic data section on earn page

here are the specs in the figma

<img width="1027" alt="Screen Shot 2021-04-12 at 12 44 59 AM" src="https://user-images.githubusercontent.com/238235/114341704-61fe8780-9b28-11eb-9000-6cab4e9fa0f8.png">


Bounties Cards
The values should not have a hover state, it suggests that the values are clickable.
<img width="436" alt="Screen Shot 2021-04-12 at 12 46 55 AM" src="https://user-images.githubusercontent.com/238235/114341888-c7527880-9b28-11eb-8ff0-49efc05a47de.png">

Thanks!

**Comment by PixelantDesign** [comment]
@frankchen07 @octavioamu since we made the decision to update the bottom CTA's to direct to Github after jobs landing page, my recommendation is to add one line of text providing context, see below

EARN
<img width="1397" alt="Screen Shot 2021-04-12 at 1 13 44 AM" src="https://user-images.githubusercontent.com/238235/114344006-0da9d680-9b2d-11eb-844d-5d83af3e4a51.png">

On the pages below not all of the CTAs require GitHub auth so the additional text to let users know that Github is next iis provided in the relevant CTA container.

LEARN
<img width="1230" alt="Screen Shot 2021-04-12 at 1 21 26 AM" src="https://user-images.githubusercontent.com/238235/114344277-86109780-9b2d-11eb-8216-5660cae247c5.png">

CONNECT
<img width="1303" alt="Screen Shot 2021-04-12 at 1 21 45 AM" src="https://user-images.githubusercontent.com/238235/114344310-94f74a00-9b2d-11eb-9c17-f02b9b74a9ef.png">

FUND
<img width="1220" alt="Screen Shot 2021-04-12 at 1 22 05 AM" src="https://user-images.githubusercontent.com/238235/114344332-9d4f8500-9b2d-11eb-8b07-0e6da0b0e457.png">

**Comment by octavioamu** [comment]
Seems now there is not data for the top earn monthly https://gitcoin.co/leaderboard?cadence=monthly&keyword=&product=bounties 
was working when deployed

**Comment by frankchen07** [comment]
Octavio's WIP PR for changes: https://github.com/gitcoinco/web/pull/8772

There are some items we need from our stakeholders for the fixes above. 

===

- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816257274 
- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816754078

**re: LEARN page, adding the quests CTA back.** This can be done, but was not included in the final design figma. We need the correct copy from @matthewcarano and the design from @PixelantDesign in order to add this change.

===

- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816261349
- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816995254
- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816270779

**re: spacing inconsistencies for the hackathon section on EARN, LEARN, CONNECT, and the FUND page.** This use case was not included in the final design figma. Currently, the content will exist in a way where there's consistent spacing in between the title, description, date, and sponsor. This is how dynamic data is handled on this section of the page. Forcing alignment means forcing container sizes, which may cause the same problem down the line. If we want this change, @PixelantDesign will have to design for edge cases on the data shown (varying sizes of title & description).

===

- [X] https://github.com/gitcoinco/web/issues/8658#issuecomment-816262962

**re: tags on the EARN page for bounties.** This was coded up as represented in the final design figma. Octavio has fixed this on the WIP PR.

===

- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816262962

**re: grant looks like an advertisement.** Again, this is what the engineers built off of the design, if @PixelantDesign sees this change as necessary then defer to her, otherwise, it will remain the same.

===

- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816268229

**re: on the CONNECT page, mid page kernel CTA leads to somewhere different than the bottom CTA.** @matthewcarano I suggested that we keep the links the same, but I'd like final confirmation from content that this makes sense, as the links were represented differently in the design.

===

- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816270779

**re: traditional matching and qf matching text description on the FUND page.** @PixelantDesign - do we have a visual that represents this better? If not, can we at the very least periodize the sentences?

===

- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816754078

**re: on the EARN page, there is only 1 grant shown.** This was how it was designed. Confirming with @PixelantDesign that this is intended. Final call on if we want to add more, or leave as is.

===

- [X] https://github.com/gitcoinco/web/issues/8658#issuecomment-816988126

**re: on the EARN page, forcing header breaks on the "Break Free" title.** @matthewcarano, this change is not advised by the engineers because of the way responsive view works. We can add breaks to the title, but it would show up fine on a specific resolution. Once we start looking at mobile views, different screen sizes, the breaks will move like dynamic text in this case, and will line break unnecessarily for different responsive views. Marking this as complete, but it's something that won't work under responsive view conditions.

===

- [X] https://github.com/gitcoinco/web/issues/8658#issuecomment-816988126

**re: double copy in hackathons box on the earn page.** This has been fixed in the WIP PR.

===

- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816988126
- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816991145
- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816995254

**re: earn page CTA capitalization / capitalization across landing pages / discord copy changes.** This was built according to design specifications. @matthewcarano - can you work with @PixelantDesign to solidify the preferred case on all CTAs across the four pages and post the finalized version to me? Button text is easy to change, it's just we want final confirmation from you guys as stakeholders on that so we don't have multiple back and forths on this change.

===

- [ ] https://github.com/gitcoinco/web/issues/8658#issuecomment-816991145

**re: testimonial on the learn page, adding a period and breaking it into 2 paragraphs.** The period has been added in the WIP PR. I believe the break can be added since it's not inline like the "break free from 9-5" issue above.

===

- [X] https://github.com/gitcoinco/web/issues/8658#issuecomment-817476928

**re: missing dynamic data.** This is already resolved - there was a temporary outage, but now it's all good.

===

https://github.com/gitcoinco/web/issues/8658#issuecomment-817490585

**re: adding github signin context.** This is part of the homepage scope, so I'll move this over to that ticket.

**Comment by matthewcarano** [comment]
@frankchen07 

[re: LEARN page, adding the quests CTA back](https://github.com/gitcoinco/web/issues/8658#issuecomment-816257274).  I added Quest CTA copy to the figma (My recommendation is to not action people to Quests, but defer to folks who have a better understanding of the numbers.)

[re: on the CONNECT page, mid page kernel CTA leads to somewhere different than the bottom CTA.](https://github.com/gitcoinco/web/issues/8658#issuecomment-816268229) I agree to remove mid-page Kernel CTA to remain consistent with the other JTBD pages.

**Comment by PixelantDesign** [comment]
>re: on the EARN page, there is only 1 grant shown. This was how it was designed. Confirming with @PixelantDesign that this is intended. Final call on if we want to add more, or leave as is.

This was meant to feel more like a feature of a grant. We selected a top grant based on Scott's recommendation. We can't control what the grant is using for an image. It's the image that  makes it look like an advertisement.

**Comment by frankchen07** [comment]
> We can't control what the grant is using for an image. It's the image that makes it look like an advertisement.

Understand. We are proceeding without change here.

---

sentence case copy and other copy changes has been updated on the Figma, and passed to the engineers.

---

> @owocki comment about adding Learn CTA

Unfortunately, we're not going to be able to add this in for the v1 version. The decision was made that Quests is not a major action on our platform. I am happy to say that I believe we're still covered, based on navigational data from GA that suggests that most of our users find quests through the "products" nav up top, which still exists.

The decision to not add the Learn CTA back in was also a combination of it adding another potential 3-4 days of design & build space, which we just don't have, especially given our timeline. Feel free to HMU if you need any further details.

**Comment by owocki** [comment]
i was not Consulted, Informed, or Approver in any discussions about whether Quests was a "major action" or not, so i'm not sure how that "decision" has any legitimacy at all.

quests are [played 500 times per day](https://metabase.gitcoin.co/question/1009) and support our H1 2021 OKRs, and i would have challenged it had i been involved.

how on earth does adding quests to the learn CTA take 3-4 days?  ive had multiple discussions with Kyle/Alisa about adding it back, each time i've been assured it will be done, and each time it keeps not happening. at this point, i'm annoyed that something that gets so much usage and supports our OKR of retaining users on the site cannot be supported, disappointed that something so simple can't be worked through our "process", and ive lost faith that the process is working; so ill just commit the changes myself to stable when this is live (which will take about 10 minutes, and will not take 3-4 days) and go from there.

**Comment by owocki** [comment]
d8d31510b70d8f6854303a5b431d83b531263b6b this took me 10 minutes

**Comment by matthewcarano** [comment]
@owocki There's been so many content version with these pages that I can't remember when quests were on/off without looking back. But where we are now I recommended not adding it in (above) because we'll need to think through the information flow starting from the accordion down the page. It might not be enough context if we just put it in the accordion, which means we may require a new section and some design work. Adding the CTA has a bit of a ripple effect. If we do it, I want to do it right. If it's important to you I'll work on in, but I was looking at the priority list and recommending off that.

**Comment by octavioamu** [comment]
I should commented here with the WIP PR just waiting on the final decisions https://github.com/gitcoinco/web/pull/8772
by the way the quest cta was already added the other day on that one.

**Comment by owocki** [comment]
quests gets plenty of usage and it creates valuable reputational data for the network (which will be useful if we ever building a musclar email marketing infrastructure than can target users by skillset) and a mechanism for platforms to creatively market themselves to our userbase; if its not clear from my past steadfast assertations that its worth the minimal overhead to put in, let me restate that i think its worth working it in.

i feel like there is too much of a culture of negotiating priority here and not enough pride in our work and celebration of attention to detail and playful value-add that creates utility for end users.  I take pride in the belief that going the extra mile can sometimes really make or break the UX and create a hook for end-users. thats why i put a lot of effort into small product features like gitcoin quests or the avatar builder or town square back in the day. its bc i wanted all of the diversity of humanity to be able to represent themselves on the platform and to have something to do even if they werent a superskilled developer  (or rich person) yet, thus reducing the TTV.  the theory was that millenials are always looking for cool ways to represent themselves online.  [there is a name for this playbook; come for the tool stay for the network](https://www.nfx.com/post/9-habits-world-class-startups/)    i recall getting push back that i was expecting too much / was too unfocused by pushing for the avatar builder. i just don't fucking care because i am driven by the need to go the extra mile to delight an end user and i know that reputation/social data matters for a marketplace like ours.  we can ship with both velocity and high attention to detail; we just need to build that muscle.

i am totally willing to flush things out of the airlock when they're not working and it doesnt seem like theres a viable turn around plan (Gitcoin Labs, CodeFund, Universal Bounty Explorer, Standard Bounties, Gitcoin ENS, Grants 1337 Support).  please don't mistake my comments above as a straw man version of themselves "we need to do everything and be everything to everyone".  we don't.   we don't need to keep all the things i've coded either; half of the aforementioned were my code that since been ripped out.

btw, happy to see constructive feedback about how quests could be a 10/10 experience.  so far i've heard whispers that its only a 7/10 but no one has really bothered to put together any real product research or tickets to figure out how to advance it.

**Comment by frankchen07** [comment]
all changes above in staging as of 4/20.

staging review beings now!

===

just a few quick capitalization changes I caught - let's keep sentence case consistent.

![Screen Shot 2021-04-20 at 17 27 46](https://user-images.githubusercontent.com/7516920/115479748-ee323e00-a1fd-11eb-8513-6a23a46e9403.png)

Why Quadratic Funding? -> Why quadratic funding?
Gitcoin Quadratic matching -> Gitcoin quadratic matching

**Comment by matthewcarano** [comment]
I think Quadratic Funding should be capitalized. Feels like that's the convention we always use so in our world it's a proper name. Agree on "quadratic matching".
